### PR TITLE
a bit changes

### PR DESCRIPTION
**Pull request by savkk**

### CODE REVIEW

File: java/core/page-factory-core/src/main/java/ru/sbtqa/tag/pagefactory/support/DesiredCapabilitiesParser.java [hunk 1]
```diff
@@ -19,9 +17,7 @@
     public DesiredCapabilities parse() {
         final DesiredCapabilities capabilities = new DesiredCapabilities();
 
-        capabilities.setCapability(CapabilityType.UNEXPECTED_ALERT_BEHAVIOUR, UnexpectedAlertBehaviour.IGNORE);
-
-        final String capsPrefix = "webdriver." + TagWebDriver.getBrowserName().toLowerCase() + ".capability.";
+        final String capsPrefix = "webdriver.capability.";
```
Comment (varivoda) [inline java/core/page-factory-core/src/main/java/ru/sbtqa/tag/pagefactory/support/DesiredCapabilitiesParser.java]: CRITICAL Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00115)

**Comment by varivoda** [comment]
SonarQube analysis reported 1 issue
* ![CRITICAL][CRITICAL] 1 critical

Watch the comments in this conversation to review them.

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'

**Comment by clicman** [comment]
@kosteman Is it a dead request?

**Comment by kosteman** [comment]
consumed by #34

**Pull request closed without merging**